mailbox/api/src/main/scala/org/apache/james/mailbox/scala/ScalaQuotaEvent.scala

```diff
@@ -0,0 +1,108 @@
+package org.apache.james.mailbox.scala
+
+import java.time.Instant
+import java.util.Optional
+
+import julienrf.json.derived
+import org.apache.james.core.quota.{QuotaCount, QuotaSize, QuotaValue}
+import org.apache.james.core.{Domain, User}
+import org.apache.james.mailbox.MailboxListener.{QuotaEvent => JavaQuotaEvent, QuotaUsageUpdatedEvent => JavaQuotaUsageUpdatedEvent}
+import org.apache.james.mailbox.model.{QuotaRoot, Quota => JavaQuota}
+import play.api.libs.json._
+
+import scala.collection.JavaConverters._
```

Comment:
do we allow wildcards import in scala?

Reply:
good question, I see that everywhere for function imports, it allows to have all features at hand but honestly, it's very similar to Java so I don't see any reason be more lenient about hat